sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client_models.hpp

```diff
@@ -477,12 +445,12 @@ namespace Azure { namespace Security { namespace Attestation { namespace Models
     /**
      * @brief Result of a modification.
      */
-    Azure::Nullable<PolicyModification> PolicyResolution;
+    PolicyModification PolicyResolution;
 
     /**
      * @brief The SHA256 hash of the policy object which was received by the service.
```

Comment:
Aren't SHA-256 hashes fixed length? Should this be a (possibly-`Nullable`) `std::array<uint8_t, something>`?

Reply:
They are fixed length (they're 32 bytes). That change makes sense to me, I'll make it in a separate branch though (there are already too many gratuitous changes in this PR).